src/app/score.py

```diff
@@ -54,21 +54,27 @@ def calc_scores(stats: dict, score_modulator: ScoreModulator) -> dict:
     }
 
 
+def get_score_weight_or_zero(key: str, material_type: str, score_weights: ScoreWeights):
```

Comment:
I was pondering whether to remove the ScoreWeights class completely. It is not used anywhere else and only parts of its functionality. Since you find it confusing, too, I will thus remove it :)

Reply:
agreed looks like another YAGNI :roll_eyes:

Reply:
Removed.